### PR TITLE
VBID-7103 Update latitude and longitude to doubles

### PR DESCRIPTION
**Pull request by psimmons-vistar**
* Updated api.proto and compiled proto to use doubles instead of floats
  for latitude and longitude
* Updated major version due to breaking change
* Updated path for knowledge base article about client since old
  knowledge base has been replaced
* Removed blank test

### CODE REVIEW

File: src/main/java/com/vistarmedia/api/message/Api.java [hunk 1]
```diff
@@ -103,12 +103,12 @@ public DisplayArea getDefaultInstanceForType() {
     private int bitField0_;
     // required string id = 1;
     public static final int ID_FIELD_NUMBER = 1;
-    private Object id_;
+    private java.lang.Object id_;
```
Comment (psimmons-vistar) [inline src/main/java/com/vistarmedia/api/message/Api.java]: Compiled with protoc version 2.4.1 and command protoc --proto_path=src/main --java_out=src/main/java src/main/protobuf/api.proto

This matched the version of proto defined in the pom.xml, so I'm not quite sure why it added explicit paths to Object.  Wanted to bring this up and see if I should be doing something different

File: pom.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
   <groupId>com.vistarmedia</groupId>
   <artifactId>client-api</artifactId>
-  <version>1.5-SNAPSHOT</version>
+  <version>2.0-SNAPSHOT</version>
```
Comment (psimmons-vistar) [inline pom.xml]: I need some guidance on the deployment procedure for this repository when this is merge-ready

File: src/main/java/com/vistarmedia/api/message/Api.java (hunk 1 above)
Comment (chrismcbride) [inline src/main/java/com/vistarmedia/api/message/Api.java]: That command looks correct to me. I imagine this is fine.

**Comment by chrismcbride** [comment]
:+1:

**Comment by markchadwick** [comment]
:+1:

**Comment by psimmons-vistar** [comment]
Version is deployed to our Maven repo and have QA'd the lat/lon changes against dev using the client.
Also communicating with Christian about putting a small note in the KB article.

Can somebody merge this for me? I don't have write access

**Pull request merged**